manifest_uaaextra.yml

```diff
@@ -2,6 +2,7 @@
 applications:
 - name: uaaextra
   host: account
+  domain: ((domain)) # needed because we split the app and admin domains
```

Comment:
with this change, we _should_ be able to drop the second file for prod, too, but I want to try it first to be sure